### DISCUSSION TITLE
Revert changes to alternative playee layout

### DISCUSSION THREAD

**Discussion by Silther** [libre-tube/LibreTube#5118] [General]
Revert changes to the alternative player layout (#5031).
Discussed in #5115.
(Just wanted to move the discussions away from the issue section)

**Comment by pokegamer5547** [comment]
On tablets maybe  it's ok but for most regular phones, the new layout makes it harder to see the upload dates of videos :(

**Reply by JeremyStarTM** [reply]
Yeah, it's bad on phones.

**Reply by Silther** [reply]
> Yeah, I hate it already.

please be polite
and remember, bynro is developing this app in his spare time.

**Comment by DanielProg39** [comment]
IMHO there's no critical difference between the old and new layouts. I agree that the old one looks more familiar and is a bit more useful, but for me the new layout is better because the old one and the one in the YT app are too eye-catching and designed to make the user want to interact with it. That's not surprising, given the addictive nature of how apps are designed these days. The new layout offers more deliberate control, as it's smaller and a little less intuitive, but still quite useful.

**Reply by Bnyro** [reply]
<3

**Comment by RuthlessRuler** [comment]
I think user should be given this option to choose the layout. 
Phones are generally tall, so vertical scrolling makes more sense here, and avoids blank space. 

With the current layout, the video title is always cut off making you interact (long press/click the video) with the video on phone.

Additionally, I think it makes more sense for accessibility too. I have higher eye power and it makes it harder for me to read thumbnails if I am keeping the device at a distance.

Would highly appreciate if an option of layout could be added back.

**Reply by Tetzuya75** [reply]
I agree with you.
in any cases, vertical scrolling is more user-friendly on android system (smartphone & tablet)

**Comment by crisinho** [comment]
Second this. On my tablet I used to have two rows of related videos shown on screen. Now I only see one small row (same size) with blank space below it where the other row used to be. So essentially I only see half the content and have to scroll even further. Also, the horizontal scrolling doesn't feel intuitive to me when virtually all scrolling on a phone is based on vertical lists (this app not being an exception, everything from the playlists to the subscriptions is being scrolled vertically).

While having less unnecessary options is generally good, I'd argue in this case where it is such a departure from well known vertical scrolling, it's not an option that should be omitted. I could live with this being the preconfigured Layout but I'd love to have the option, especially if it's less efficient to find something this way.

**Reply by DanielProg39** [reply]
I generally agree. The pure reason for the change was to make the layout more suitable for tablets. If it's now even less comfortable on tablets (so much so that it doesn't display correctly), I don't see why we should stick to it.

**Comment by Snailhail** [comment]
I'm the same I can't adapt to It vertical, but see why they added It. I'm thankful for the work everyone has done for this app. For now I'll  make do intill a proper fix comes.

**Comment by Handrail9** [comment]
Is there truly and genuinely a reason to completely remove the option when you could have just changed the default behavior? If this isn't reverted and the option doesnt come back I plan on sticking to 0.19 until it breaks, and if/when it does moving back to NewPipe or a better alternative if one becomes available.
